### DISCUSSION TITLE
项目发布到外网

### DISCUSSION THREAD

**Discussion by keng666** [illacloud/illa-builder#785] [Q&A] [answered]
这个工具真的很棒。
请问如何将做的项目打包成一个独立网站发布到云服务器？
我发现别人访问项目发布后的网址时候也要登录illa。
刚入门，如有描述不正确请指正。
感谢！

**Comment by AruSeito** [comment]
您好，目前是不支持将做好的项目发布到云服务器的。
发布的项目目前是仅支持自己团队内的成员访问的，后续会进行相关功能优化。
如果您有该方面的需求可以跟我们的产品经理进行沟通。 @wenzhuo2022

**Comment by wenzhuo2022** [comment, marked as answer]
这个功能接下来几周会做支持❤️
